packages/amplify-category-auth/src/index.js

```diff
@@ -312,6 +318,24 @@ async function executeAmplifyCommand(context) {
   await commandModule.run(context);
 }
 
+/**
+ * Entry point for headless commands
+ * @param {any} context The amplify context object
+ * @param {string} headlessPayload The serialized payload from the platform
+ */
+const executeAmplifyHeadlessCommand = async (context, headlessPayload) => {
+  switch (context.input.command) {
+    case 'add':
+      if (projectHasAuth(context)) return;
```

Comment:
nit: can we have block here, which increases readability? also ```switch...break``` is better for understanding than returning straight from a case block IMHO

Reply:
Added the block.

I like early returns because when reading the function you know that's the end of an execution path; you don't have to look any further to see if there's other stuff